src/js/context/index.ts

```diff
@@ -1,34 +1,3 @@
-import { JSONSchemaFormDataProvider } from "./json_schema_provider";
-import {
-    ApplicationContextMixin,
-    JobContextMixin,
-    MaterialContextMixin,
-    MaterialsContextMixin,
-    MaterialsSetContextMixin,
-    MethodDataContextMixin,
-    WorkflowContextMixin,
-} from "./mixins";
 import { JobContextPickKeysForMixin, WorkflowContextPickKeysForMixin } from "./pickers";
-import { ContextProvider } from "./provider";
-import {
-    ContextProviderRegistryContainer,
-    createAndPatchRegistry,
-    extendAndPatchRegistry,
-} from "./registry";
 
-export {
-    ContextProvider,
-    ContextProviderRegistryContainer,
-    extendAndPatchRegistry,
-    createAndPatchRegistry,
-    JobContextPickKeysForMixin,
-    JSONSchemaFormDataProvider,
-    WorkflowContextPickKeysForMixin,
-    ApplicationContextMixin,
-    MaterialContextMixin,
-    MaterialsContextMixin,
-    MaterialsSetContextMixin,
-    MethodDataContextMixin,
-    JobContextMixin,
-    WorkflowContextMixin,
-};
+export { JobContextPickKeysForMixin, WorkflowContextPickKeysForMixin };
```

Comment:
We should move these to wode too

Reply:
I noticed that we use both of them in web-app only. Do we even need to store it somewhere outside the web-app?